app.rb

```diff
@@ -0,0 +1,107 @@
+require './person'
+require './book'
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+#rubocop:disable all
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is quality and clean.